src/index.css

```diff
@@ -0,0 +1,191 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/cyptobg.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+body::before {
+  display: block;
+  content: '';
+  height: 110px;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+a {
+  text-decoration: none;
+}
+
+.autoimg {
+  padding: 9px 0 9px 0;
+}
+
+.header {
+  display: flex;
+  position: fixed;
+  top: 0;
+  background-color: #002855;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.homepage,
+.center,
+.title {
+  color: #b1cb32;
+}
+
+.url {
+  color: #adb5bd;
+  text-decoration: underline;
+}
+
+.url:hover {
+  text-decoration: none;
+}
+
+.btn-con {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+}
+
+.input {
+  display: flex;
+  justify-content: center;
+  margin: 15px 0 15px 0;
+}
+
+input {
+  width: 60%;
+  padding: 10px;
+  border: none;
+  border-radius: 10px;
+  font-size: 25px;
+  background-color: silver;
+  color: black;
+}
+
+.center {
+  text-align: center;
+}
+
+.break {
+  width: 50%;
+}
+
+.flex {
+  background-color: #001d3d;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 15px;
+  margin: 4px;
+  border-radius: 8px;
+}
+
+.main-container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.animate {
+  border: 10px solid #f3f3f3;
+  border-top: 10px solid #3498db;
+  position: relative;
+  border-radius: 50%;
+  margin: auto;
+  margin-bottom: 10px;
+  width: 80px;
+  height: 80px;
+  animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
```

Comment:
- [ ] As said by the previous reviewer, Kindly make sure your layout looks similar to the provided design.
`Hint: You may consider using the grid container this way:`
```css
.grid {
  display: grid;
  padding-top: 0;
  grid-template-columns: repeat(2, 1fr);
}
.element-in-the-grid:nth-child(4n + 2),
.element-in-the-grid:nth-child(4n + 3) {
  background-color: rgba(0, 0, 0, 0.08);
}

.element-in-the-grid:nth-child(4n - 4),
.element-in-the-grid:nth-child(4n + 1) {
  background-color: #eb4c89;
}
```

Reply:
Good to hear from @christianbiring1, thanks for the feedback but I'm confused about the first feedback because I'm already using flexbox for the header

Reply:
But nothing was changed on your nav bar since the last reviewer suggested the changes. You may consider checking them and redeploy your app to include those changes.